.github/workflows/deploy.yml

```diff
@@ -0,0 +1,50 @@
+name: "Deploy to gh-pages when PR is merged"
+on:
+  pull_request:
+    # Deploy after merging PRs to master
+    types: closed
+    branches:
+      - master
+  schedule:
+    # Deploy hourly between 9am and 7pm on weekdays
+    - cron: "0 9-19 * * 1-5"
```

Comment:
While the existing job may do this, I'm confused why a periodic deploy is necessary. Could you help me understand why we need it for GitHub Pages?

Reply:
Looks like removing the cron trigger would also simplify the steps vvv.

Reply:
Because the dev docs pull in external content (`docs/` folders for various repos, some hardcoded `ExternalDoc.fetch` references, govuk-content-schemas to name a few). Having a periodic rebuild ensures the docs are always no more than an hour out of date.